shotover-proxy/Cargo.toml

```diff
@@ -63,7 +63,7 @@ halfbrown = "0.1.11"
 
 # Transform dependencies
 redis-protocol = { version = "4.0.1", features = ["decode-mut"] }
-cassandra-protocol = { git = "https://github.com/krojew/cdrs-tokio" }
+cassandra-protocol = { git = "https://github.com/krojew/cdrs-tokio", rev = "dd564cbb6e348952c8b8a9c5c87ff4f98c28077e" }
```

Comment:
Why are we pinning here? was there a regression in a later commit?

If you just want to pin here so that a `cargo update` wont hit breaking changes, then thats fine but just wanted to understand the reasoning.

Reply:
A quick test reveals that yes the latest commit does have breaking changes in it.